ui/desktop/app/routes/scopes/scope/projects/sessions/index.js

```diff
@@ -105,34 +105,11 @@ export default class ScopesScopeProjectsSessionsIndexRoute extends Route {
     const totalItems = sessions.meta?.totalItems;
 
     // Query all sessions and all targets for defining filtering values if entering route for the first time
-    if (from !== 'scopes.scope.projects.sessions.index') {
```

Comment:
I changed the check but the comment is actually accurate now, it was previously wrong since it would always query when routes are changed, just not for the same route

Reply:
If a user changes the scope, say from global to a different org, would `this.allSessions` and `this.allTargets` get called again?

Reply:
No it wouldn't

Reply:
I think that might change which projects are displayed in the scopes dropdown. I will pull these changes and test.

Reply:
That display happens from pulling projects from the `modelFor('scopes.scope.projects')` which we do an intersection on

Reply:
Great news! I was completely wrong. Working great!